redaxo/src/addons/mediapool/lib/media.php

```diff
@@ -87,6 +87,21 @@ public static function get($name)
         });
     }
 
+    /**
+     * @throws rex_sql_exception
+     * @return null|static
+     */
+    public static function getById(int $mediaId)
```

Comment:
Ich würde es glaube ich eher `forId()` nennen, weil es dann besser zu den methoden passt, die wir zuletzt eingeführt haben (`rex_template/rex_module::forKey` und `rex_user::forLogin`). `getByX`-Methoden haben wir eigentlich nicht mehr.

```suggestion
    public static function forId(int $mediaId): ?self
```

Reply:
ist nun